package/boot/uboot-envtools/files/ipq40xx

```diff
@@ -76,6 +76,9 @@ linksys,whw03v2)
 zyxel,nbg6617)
 	ubootenv_add_uci_config "/dev/mtd6" "0x0" "0x10000" "0x10000"
 	;;
+zyxel,wsq50)
+	ubootenv_add_uci_config "/dev/mtd2" "0x0" "0x10000" "0x10000"
```

Comment:
Instead of this, you can just add `compatible = "u-boot,env";` to the partition that is holding the env.